### ISSUE TITLE
Readme wrong?

### ISSUE THREAD

**Issue by nikue** [CodingZeal/padlock#4]
I think:

gem 'padlock'

should be:

gem 'padlock-rails'

no? Otherwise it installs an earlier version 

Also, generator doesn't seem to work properly with given instruction. Had to copy the migration over manually.

**Comment by acuppy** [comment]
Hey @nikue!  Thanks for bringing that up.  I'll take a look.

**Comment by nikue** [comment]
Not at all. Also I get a "no method" error for acts_as_padlock_user

**Comment by jrex** [comment]
Did this ever get resolved @nikue ? I'd absolutely love to use this gem as the capabilities are exactly what I'm looking for however it just doesn't seem to work. I cannot for the life of me get it to integrate with ActiveRecord::Base nor ApplicationRecord (Rails 5). It seems that:

- It needs to be "gem 'padlock-rails'" for it to get the latest version 
- The generator for the migration doesn't work
- The integration with ActiveRecord::Base does not work for a Rails 5 project
- The functionality of the module itself from the Readme doesn't seem to work... basic things like setting the config in an initializer using "Padlock.config.timeout = 1.week" doesn't work. I put this into a config/initializers/padlock.rb file as opposed to just a config/padlock.rb file as the readme specified.
- Another piece of functionality that I can't seem to get working from the Readme is: Padlock.locked?(Blah.find(1)) which comes back with an error: 'NoMethodError: undefined method 'locked?' for #<Blah:0x000000058dcdb8>"
- Even attempting to throw the code into the lib directory with some tweaking couldn't get it to work

I'm a little stumped ... @acuppy  are these currently known issues or do I have some sort of FrankenIssue? I'd love to be able to use this gem but seem to be up against a wall here currently.

**Comment by nikue** [comment]
No, there's a lot that needs updating for Rails 5+ (maybe even Rails 4?). I
only needed a very small part of the gem's functionality, so I rolled my
own instead.

Nikue Gardner

**Comment by jrex** [comment]
That might be the fastest response ever to a comment! 

That's definitely disappointing... I guess I'll just roll my own code as well then.
